hr_cae/i18n/fr.po

```diff
@@ -25,49 +25,6 @@ msgstr "Code APE"
 msgid "Administrative"
 msgstr "Administratif"
 
-#. module: hr_cae
```

Comment:
If I understand it correctly all these lines are dropped because they correspond to fields that were moved to a different module. If not to much work, it would be cool to add them there (although I think we're managing the translations mostly via the interface currently).